### PR TITLE
🐛 fix: set tooltip arrow and custom tooltip in center

### PR DESCRIPTION
**Pull request by ShwetaChauhan18**
🐛 fix: set tooltip arrow and custom tooltip in center

### CODE REVIEW

**Review by BirjuVachhani — APPROVED** [review]
LGTM

File: lib/src/tooltip_widget.dart [hunk 1]
```diff
@@ -217,57 +232,106 @@ class _ToolTipWidgetState extends State<ToolTipWidget> {
...
+                                constraints: BoxConstraints(
+                                  maxWidth: widget.screenSize.width - 42,
+                                ),
+                                child: Column(
+                                  crossAxisAlignment:
+                                      _coords != null && _coords!.isArrowCenter
+                                          ? CrossAxisAlignment.center
+                                          : CrossAxisAlignment.start,
+                                  children: [
+                                    Container(
```
Comment (vatsaltanna) [inline lib/src/tooltip_widget.dart:291]: use padding

File: lib/src/tooltip_widget.dart [hunk 2]
```diff
@@ -259,57 +274,106 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
...
+                                      padding: widget.contentPadding,
+                                      child: Column(
+                                        crossAxisAlignment: widget.title != null
+                                            ? CrossAxisAlignment.start
+                                            : CrossAxisAlignment.center,
+                                        children: <Widget>[
+                                          widget.title != null
+                                              ? Text(
+                                                  widget.title!,
+                                                  key: _key1,
```
Comment (vatsaltanna) [inline lib/src/tooltip_widget.dart:301]: the naming convention can be improved

File: lib/src/tooltip_widget.dart [hunk 3]
```diff
@@ -259,57 +274,106 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
...
+                                                                  color: widget
+                                                                      .textColor,
+                                                                ),
+                                                              ),
+                                                )
+                                              : SizedBox.shrink(),
+                                          widget.description != null
+                                              ? Text(
+                                                  widget.description!,
+                                                  key: _key2,
```
Comment (vatsaltanna) [inline lib/src/tooltip_widget.dart:318]: the naming convention can be improved

File: lib/src/tooltip_widget.dart [hunk 4]
```diff
@@ -259,57 +274,106 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
...
+                                    if (widget.actions != null)
+                                      SizedBox(
+                                        height: widget
+                                                .spaceBetweenTitleAndButtons ??
+                                            5,
                                       ),
-                                    ],
-                                  ),
+                                    if (widget.actions != null)
+                                      LayoutBuilder(
```
Comment (vatsaltanna) [inline lib/src/tooltip_widget.dart:341]: do we need LayoutBuilder here?

File: lib/src/tooltip_widget.dart [hunk 5]
```diff
@@ -259,57 +274,106 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
...
                                 ),
                               ),
                             ),
                           )
-                        : widget.container!,
-                    if (widget.actions != null) widget.actions!
+                        : Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: [
+                              widget.container!,
```
Comment (vatsaltanna) [inline lib/src/tooltip_widget.dart:368]: widget.container should not be forced unwrap

File: lib/src/tooltip_widget.dart [hunk 6]
```diff
@@ -259,57 +274,106 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
...
+                                                )
+                                              : SizedBox.shrink(),
+                                          widget.description != null
+                                              ? Text(
+                                                  widget.description!,
+                                                  key: _key2,
+                                                  style: widget.descTextStyle ??
+                                                      Theme.of(context)
+                                                          .textTheme
+                                                          .subtitle2!
```
Comment (vatsaltanna) [inline lib/src/tooltip_widget.dart:322]: do not force unwrap

File: lib/src/tooltip_widget.dart (hunk 5 above)
Comment (ShwetaChauhan18) [reply lib/src/tooltip_widget.dart:368]: in this case it will not null as we already added condition

File: lib/src/tooltip_widget.dart (hunk 4 above)
Comment (ShwetaChauhan18) [reply lib/src/tooltip_widget.dart:341]: we are calculating width based on render view that's why I used but without it it's working so removed.

**Review by vatsaltanna — APPROVED** [review]

**Pull request merged**